src/fe-text/irssi.c

```diff
@@ -74,11 +75,20 @@ void mainwindow_activity_deinit(void);
 void mainwindows_layout_init(void);
 void mainwindows_layout_deinit(void);
 
+#ifdef HAVE_SECCOMP
```

Comment:
I am not familiar with how irssi prefers to do, but it is generally considered better to have functions like this be defined no matter what, and then the function bodies themselves can be guarded. Doing so limits the amount of `#ifdef`s you need to scatter across the code.